front/app/components/PostShowComponents/Comments/Comment/CommentVote/UpvoteButton.tsx

```diff
@@ -42,41 +41,35 @@ const ButtonWrapper = styled.button`
   border: none;
   cursor: pointer;
 
-  &.disabled {
-    cursor: auto;
-  }
-
-  &.enabled:not(.voted):hover {
-    color: #000;
-
-    ${UpvoteIcon} {
-      fill: #000;
-    }
-  }
-
-  &.enabled.voted {
+  &.enabled.voted,
+  &.disabled.voted {
     color: ${colors.success};
 
     ${UpvoteIcon} {
       fill: ${colors.success};
     }
   }
 
+  &.enabled:not(.voted),
   &.disabled:not(.voted) {
-    color: ${lighten(0.25, colors.textSecondary)};
+    color: ${colors.textSecondary};
 
     ${UpvoteIcon} {
-      fill: ${lighten(0.25, colors.textSecondary)};
+      fill: ${colors.textSecondary};
     }
   }
 
-  &.disabled.voted {
-    color: ${lighten(0.25, colors.success)};
+  &.enabled:not(.voted):hover {
+    color: #000;
```

Comment:
Minor. Same for line 66
```suggestion
    color: ${colors.black};
```